### ISSUE TITLE
History grid follow-up

### ISSUE THREAD

**Issue by bgruening** [galaxyproject/galaxy#17283]
In https://github.com/galaxyproject/galaxy/pull/17219 @guerler migrated the grid to vuejs. This issue is a follow-up with items that we have found during the review but are not strictly due to the PR but long-standing problems.

* history import button is missing in the expanded/advanced search
* advanced search looks misaligned ↓

![grafik](https://github.com/galaxyproject/galaxy/assets/469983/2d124fad-1fa3-4c01-b21d-71a370cdefe6)

* the advanced search is also not aligned to other pages, e.g. tools, where it is part of the side-panel
* renaming of histories should be inline and not require a new page
* grid header should be fixed and not disappear if you scroll
* delete and delete permanently should not be listed both at the same time in the drop-down, it should be a two-step process
* it would be nice to integrate a link to the multi-history view from that page
* the count column don't line up

![grafik](https://github.com/galaxyproject/galaxy/assets/469983/f33d0b44-714f-4281-a43d-936dbeca58e3)

**Comment by ahmedhamidawan** [comment]
> I think I'd prefer having them in the center panel

I agree, advanced searches that show an extensive list of results in the center panel, should live _in_ the center panel (like in the recently vueified grids; like the upcoming new `WorkflowList` view or the pages/histories... views). The only case for having an advanced search in a panel is if there is a list of items below the search _that is filterable_; like in the `HistoryPanel` where the filters actually apply to the items below the search.

In the case of the Tool search, since the filters from the advanced menu do not apply to the tools below the search, the panel should not have an advanced search, and instead the `FilterMenu` component should be moved to the center panel `ToolsList` (like in all the grids). There then has to be a button that points to the `ToolsList` in the `ToolPanel` though, which needs to be prominent.
